packages/core/contracts/cross-chain-oracle/GovernorSpoke.sol

```diff
@@ -10,12 +11,12 @@ import "../common/implementation/Lockable.sol";
  */
 contract GovernorSpoke is Lockable, ChildMessengerConsumerInterface {
     // Messenger contract that receives messages from root chain.
-    ChildMessengerConsumerInterface public messenger;
+    ChildMessengerInterface public messenger;
 
     event ExecutedGovernanceTransaction(address indexed to, bytes data);
     event SetChildMessenger(address indexed childMessenger);
 
-    constructor(ChildMessengerConsumerInterface _messengerAddress) {
+    constructor(ChildMessengerInterface _messengerAddress) {
```

Comment:
Nit: We don't even call any functions on this `messenger`, should we just store it as an `address`?

Reply:
i personally like storing things as their interface as it makes it more explicit.